.github/release_template.md

```diff
@@ -0,0 +1,26 @@
+<!--
+This is the template for new release issues.
+-->
+
+Release a new version
+## TASKS:
+- [ ] [GIT] Create branch release_3.5.4 (freeze the code)
+- [ ] [GIT] Create branch library release_ios_app_3.5.4 (freeze the code)
+- [ ] [DIS] Update changelog ios app
+- [ ] [DEV] Update version number 3.5.4
```

Comment:
should the release template indicate a version id? maybe the template should have a generic value, and when the release issue is created, set some of the tasks with the particulars of the release